website/docs/enterprise/oidc.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: OIDC Integration
+sidebar_position: 6
+---
+
+import TierLabel from "../_components/TierLabel";
+
+<h1>
+  {frontMatter.title} <TierLabel tiers="enterprise" />
+</h1>
```

Comment:
Avoid this pattern for now, it doesn't work well with pasting links into slack etc.

```suggestion
# OIDC Integration <TierLabel tiers="enterprise" />
```